src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1306,6 +1312,55 @@ private async Task<bool> OnAssemblyLoadedJSEvent(SessionId sessionId, JObject ev
             }
         }
 
+        private async Task OnSetEntrypointBreakpoint(SessionId sessionId, JObject args, CancellationToken token)
+        {
+            try
+            {
+                ExecutionContext context = GetContext(sessionId);
+
+                var argsNew = JObject.FromObject(new
+                {
+                    callFrameId = args?["callFrames"]?[0]?["callFrameId"]?.Value<string>(),
+                    expression = "assembly_name_str + '|' + entrypoint_method_token",
+                });
+                Result assemblyAndMethodToken = await SendCommand(sessionId, "Debugger.evaluateOnCallFrame", argsNew, token);
+
+                var assemblyAndMethodTokenArr = assemblyAndMethodToken.Value["result"]["value"].Value<string>().Split('|');
```

Comment:
nit: check that the array length==2, and the parts are not empty. If not, then log a message with the string that we received, because this might indicate that the proxy, and the runtime are mismatched.

nit: Also, use `StringSplitOptions.TrimEntries`, just to be safe:)

Reply:
Maybe even just always log the received string as a debug message.